pkg/vault/contracts/Vault.sol

```diff
@@ -646,7 +646,7 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
             bptAmountOut = 0;
             amountsInScaled18 = maxAmountsInScaled18;
         } else if (params.kind == AddLiquidityKind.UNBALANCED) {
```

Comment:
not strictly related with this PR, but what about donations? Those are also a kind of unbalanced liquidity.

Reply:
Not sure what you're asking here. We have separate types and flags for donations and regular add unbalanced, and the flags are specific to the operation.